### PR TITLE
tests: download objecter_requests direct from github

### PR DESCRIPTION
**Pull request by smithfarm**
Fixes: http://tracker.ceph.com/issues/18393
Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

**Comment by smithfarm** [comment]
WARNING: master commit has not yet been merged!

File: qa/suites/powercycle/osd/tasks/admin_socket_objecter_requests.yaml [hunk 1]
```diff
@@ -10,4 +10,4 @@ tasks:
 - admin_socket:
     client.0:
       objecter_requests:
-        test: "http://git.ceph.com/?p={repo};a=blob_plain;f=src/test/admin_socket/objecter_requests;hb={branch}"
+        test: "http://github.com/ceph/ceph/tree/master/src/test/admin_socket/objecter_requests"
```
Comment (dachary) [inline qa/suites/powercycle/osd/tasks/admin_socket_objecter_requests.yaml]: s/master/branch/ ?

**Comment by dachary** [comment]
I don't think this is needed in master.

**Comment by smithfarm** [comment]
Testing hammer-only fix at #12759

**Comment by smithfarm** [comment]
ah, I didn't see https://github.com/ceph/ceph/commit/bcbe45d9 - that's why it's not needed.

**Pull request closed without merging**